content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -374,6 +374,12 @@ const prisma = new PrismaClient({
 
 ### <inlinecode>rejectOnNotFound</inlinecode>
 
+<Admonition type="info">
+
+**Deprecated:** `rejectOnNotFound` is deprecated in v4.0.0. From v4.0.0, use the queries [`findUniqueOrThrow`](#inlinecodefinduniqueorthrowinlinecode) or [`findFirstOrThrow`](#inlinecodefindfirstorthrowinlinecode).
```

Comment:
Maybe worth noting that it's also scheduled for removal in 5.0.0

Reply:
For the sake of docs maintainability, we'll omit this information (the version might change, and then we'll likely end up with incorrect information in the docs)